pyproject.toml

```diff
@@ -176,6 +176,7 @@ ignore = [
     # unknown if this is actually an issue
     "RUF005", # might not be actually faster
     "RUF006", # might not be an issue/very extreme cases
+    "PLC1901", # somewhat buggy, false positives, incorrect suggestions, and whatnot
```

Comment:
I didn't see any false positives or incorrect suggestions with this, it should be fine to add.

Reply:
Looked closer to ours. In our case, both of the errors are not false positives, but I was also checking the issue tracker which has multiple reports.

Reply:
https://github.com/DisnakeDev/disnake/pull/985/commits/39af22e3d8eb710132baabf4aed87920378143cd 

I should probably add a changelog entry tbh...